src/Analyzer.Core/TemplateAnalyzer.cs

```diff
@@ -114,47 +116,12 @@ private static string LoadRules()
         }
 
         /// <summary>
-        /// Loads a configurations file. If no file was passed, checks the default directory for this file.
-        /// </summary>
-        /// <param name="configurationsFilePath">The path to a configuration file to read.</param>
-        /// <returns>Configuration file path contents if a file exists.</returns>
-        private string GetConfigurationFileContents(FileInfo configurationsFilePath)
-        {
-            try
-            {
-                string path;
-                if (configurationsFilePath != null)
-                {
-                    path = configurationsFilePath.FullName;
-                }
-                else
-                {
-                    path = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),
-                        "Configurations", "Configuration.json");
```

Comment:
Where is this implemented? I don't see it in Cli.CommandLineParser.ReadConfigurationFile()